frontend/src/components/SignUp.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+const SignUp = () => {
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    console.log("Form submitted");
+  };
+
+  return (
+    <div>
+      <form
+        method="post"
+        action="/st21-what-the-wine/public/index.html"
+        onSubmit={handleSubmit}
+      >
+        <fieldset>
+          <legend>Sign Up for personalized Wine Recommendations</legend>
+          <p>
+            <label name="firstname">First Name</label>
+            <input type="text" name="firstname"></input>
```

Comment:
`name` is only for inputs, what you can pass to `label` elements is the `for` attribute (in React we use `htmlFor` because `for` is a reserved word in JS, but it's gonna be rendered as `for` ) and the value of the attribute needs to match the value of the `id` attribute passed to the target `input` that we want to link to the label.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form